plugins/mcreator-core/procedures/entity_get_saturation.json

```diff
@@ -0,0 +1,26 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    }
+  ],
+  "output": "Number",
+  "colour": "%{BKY_MATH_HUE}",
+  "mcreator": {
+    "toolbox_id": "playermanagement",
+    "toolbox_init": [
+      "<value name=\"entity\"><block type=\"entity_from_deps\"></block></value>"
+    ],
+    "inputs": [
+      "entity"
+    ],
+    "dependencies": [
```

Comment:
Where is this dependency used? From what I can see, you are using inputs to access the desired entity.

Reply:
If it isn't used anywhere, I suggest you remove it, since it makes little to no sense to have redundant dependencies.

Reply:
Done!

Reply:
>  no sense to have redundant dependencies.

Actually, this is not redundant but a bug as it would require dependency in cases it would not be needed.